opentech/apply/funds/templates/funds/applicationsubmission_detail.html

```diff
@@ -49,12 +49,11 @@ <h5>Your {{ object.previous.stage }} application has been accepted.</h5>
                     <h6 class="heading heading--submission-meta">
                         <span>Submitted: <strong>{{ object.submit_time.date }} by {{ object.user.get_full_name }}</strong></span>
                         <span>Last edited: <strong>{{ object.last_edit.timestamp.date }} by {{ object.last_edit.user.get_full_name }}</strong></span>
-                        {% if request.user|has_edit_perm:object and object.user == request.user %}
-                            <a href="{% url 'funds:submissions:edit' object.id %}">
-                                Edit
-                                <svg class="icon icon--pen"><use xlink:href="#pen"></use></svg>
-                            </a>
-                        {% endif %}
+                        <a class="link link--edit-submission {% if request.user|has_edit_perm:object and object.user == request.user %}is-active{% endif %}" href="{% url 'funds:submissions:edit' object.id %} ">
+                            Edit
+                            <svg class="icon icon--pen"><use xlink:href="#pen"></use></svg>
+                            <svg class="icon icon--padlock"><use xlink:href="#padlock"></use></svg>
```

Comment:
Can the icons only be injected in the relevant context?
i.e. when editable only the pen icon is added..